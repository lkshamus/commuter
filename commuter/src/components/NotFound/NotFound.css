.error-text {
  text-align: center;
}

.error-btn {
  margin: auto;
  display: flex;
  align-items: center;
  justify-content: center;
  background-color: #c4d4e0;
  color: #666;
  font: 300 18px/1.7 'Open Sans', sans-serif;
  border: 0;
  width: 305px;
  outline: none;
  height: 43px;
  border-radius: 50px;
}