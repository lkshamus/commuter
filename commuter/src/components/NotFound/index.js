import React from 'react'


const NotFound = () => {
  return(
  <div>
    <h3>the page you are requesting is not found</h3>
  </div>
  );
};

export default NotFound;