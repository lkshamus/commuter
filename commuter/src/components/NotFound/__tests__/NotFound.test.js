/* eslint-disable */
import React from 'react';
import NotFound from '../index';
import { shallow } from 'enzyme';

describe('NotFound', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<NotFound />)
    expect(wrapper).toMatchSnapshot()
  })
})