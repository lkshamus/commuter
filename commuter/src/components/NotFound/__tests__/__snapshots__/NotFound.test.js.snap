// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h3>
        the page you are requesting is not found
      </h3>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "the page you are requesting is not found",
      },
      "ref": null,
      "rendered": "the page you are requesting is not found",
      "type": "h3",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h3>
          the page you are requesting is not found
        </h3>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "the page you are requesting is not found",
        },
        "ref": null,
        "rendered": "the page you are requesting is not found",
        "type": "h3",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
